libs/sdk-ui-tests-e2e/cypress/tools/attributeFilterLimit.ts

```diff
@@ -0,0 +1,42 @@
+// (C) 2024 GoodData Corporation
+export class AttributeFilterLimit {
+    getElement() {
+        return cy.get(".attribute-filter__limit__popup");
+    }
```

Comment:
we should add a blank line between methods so that it's easy to read, please also fix for below methods